specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/examples/SuccessfulSentimentRequest.json

```diff
@@ -0,0 +1,127 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "language": "en",
+          "id": "1",
+          "text": "Hello world. This is some input text that I love."
+        },
+        {
+          "language": "en",
+          "id": "2",
+          "text": "It's incredibly sunny outside! I'm so happy."
+        },
+        {
+          "language": "en",
+          "id": "3",
+          "text": "Pike place market is my favorite Seattle attraction."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "id": "1",
+            "sentiment": "positive",
+            "documentScores": {
+              "positive": 0.998519241809845,
+              "neutral": 0.0009635657188483,
+              "negative": 0.000517153472174
+            },
+            "sentences": [
+              {
+                "sentiment": "neutral",
+                "sentenceScores": {
+                  "positive": 0.070910170674324,
+                  "neutral": 0.9124033451080322,
+                  "negative": 0.0166865326464176
+                },
+                "offset": 0,
+                "length": 12,
+                "warnings": []
+              },
+              {
+                "sentiment": "positive",
+                "sentenceScores": {
+                  "positive": 0.998519241809845,
+                  "neutral": 0.0009635657188483,
+                  "negative": 0.000517153472174
+                },
+                "offset": 13,
+                "length": 36,
+                "warnings": []
+              }
+            ],
+            "warnings": []
+          },
+          {
+            "id": "2",
+            "sentiment": "positive",
+            "documentScores": {
```

Comment:
* documentScores will be confidenceScores
* sentenceScores will be confidenceScores
* outputs are rounded to 2 decimal places. 
* The sentence will be returned in the `sentences` array:
For example 
````
"sentences": [
{
  "text": "some sentence text."
  "sentiment" : "neutral"
  "confidenceScores" : {
....
}

Reply:
Sentiment will return the text as well for each sentence.

Reply:
The confidence scores change is relevant for all endpoints, except for language detection which we need to discuss with the board. Technically, LD will be a breaking change for customers since it's the only endpoint that remains as-is from v2.1 (although it means we'll be inconsistent).

Reply:
I have updated the score to confidenceScore everywhere except LD. @josuhazure can you look into the above comment of text included for each sentence as I am not aware of the contract itself

Reply:
@assafi / @laramume  / @ctufts / @suhas92 , I applied adding sentence text and changed the documentScores to confidenceScores to BOTH 3.0 and 3.1. 
Please review again and let me know.